### PR TITLE
Power LED color and function

### PR DESCRIPTION
**Pull request by namiltd**
Power LED color and function for all platforms

### CODE REVIEW

**Comment by neheb** [comment]
```
 Author name (namiltd) need to be your real name 'firstname lastname'
Commit subject line MUST start with '<area>: ' (POWER LED color and function)
Signed-off-by is missing or doesn't match author (should be 'Signed-off-by: namiltd <namiltd@users.noreply.github.com>')
```

what a PR...

Requires changing 01_led files too.

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-wr-1.dts [hunk 1]
```diff
@@ -36,7 +36,8 @@
 		pinctrl-names = "default";
 
 		led_power: power {
-			label = "blue:power";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_POWER;
```
Comment (hnyman) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-wr-1.dts:40]: There are at least some unnecessary changes, creating duplication, as the items already exist:

```
		led_power: power {
-			label = "blue:power";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_POWER;
			gpios = <&tlmm 0 GPIO_ACTIVE_HIGH>;
			color = <LED_COLOR_ID_BLUE>;
			function = LED_FUNCTION_POWER;
```

File: target/linux/mediatek/dts/mt7986b-netgear-wax220.dts [hunk 2]
```diff
@@ -43,7 +43,8 @@
 
 		led_power_amber: power_amber {
 			gpios = <&pio 15 GPIO_ACTIVE_LOW>;
-			label = "amber:power";
+		  color = <LED_COLOR_ID_AMBER>;
+		  function = LED_FUNCTION_POWER;
```
Comment (csharper2005) [inline target/linux/mediatek/dts/mt7986b-netgear-wax220.dts:47]: Wrong indent here and some other files.

**Comment by csharper2005** [comment]
Have you already made run tests? Shouldn't we add `#include <dt-bindings/leds/common.h>` to the dts?
